src/front/app/components/Challenges/ChallengeList.js

```diff
@@ -1,7 +1,49 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import actions from './../../actions/challenges';
+import Challenge from './Challenge';
+import TopChallenge from './TopChallenge';
 
-export default () => (
-    <div>
-        ChallengeList
-    </div>
-);
+class ChallengeList extends React.Component {
+    constructor(props) {
+        super(props);
+        const { challenges, dispatch } = props;
+        if (!challenges.length) {
+            dispatch(getInitialChallenges());
+        }
+    }
+
+    render() {
+        return (
+            <ul className="challenges__list">
+                <TopChallenge { ...this.props.challenges[0] } />
+                { this.props.challenges.slice(1).map((challenge, index) => (
```

Comment:
Consider using destructuring assignment at the beginning of the render function:
const {challanges} = this.props;